### PR TITLE
Fixed layout problems

### PR DESCRIPTION
**Pull request by alvarengathomas**
This PR includes fixes for most of the problems @dvnahuel requested in the following link:
https://www.figma.com/design/4fVd9wneclsvYDYD95ApZ9/Hemi-Portal?node-id=5546-22666&t=KHgKWmnEqCtX1I5C-1

The list below shows the fixes done in this PR:

- [X] (Landing Page) Border color is different
- [X] (Landing Page) The box is missing the shadow
- [X] (Landing Page) on-hover the background whould be grayscale/50
- [X] (Landing Page) Footer here is full width, it should only occupy the width of the cards container
- [X] (Landing Page) Border is different color and with wider border
- [X] (Landing Page) On-hover this socials should be grayscale/950
- [X] (Tools Page) Arrow icon is different from design
- [X] (Tools Page) Space between these elements should be 8px
- [X] (Tools Page) Space between this elements and the text on the top should be 16px
- [X] (Tools Page) The space between these containers should be 16px
- [X] (Tools Page) Let’s remove the footer border and adjust the margins, these should be 24px bottom left and right, bot elements should be with with space-between property
- [X] (Tools Page) Left section container should be 100% VH with 8px of margin
- [X] (Tools Page) The header should be with the same margins as in the landing page (20px top, 24 right)
- [X] (Tools Page) Background color here should be white
- [X] (Tools Page) We should use validations, at least native ones in this case, ‘CLAIM’ should be disabled until the user inputs an ID

The pending fixes are:
- [ ] (Tools Page) Let’s use rainbow kit for wallet management 
  > It will require a big refactor to change wallet management to rainbow kit

- [ ] (Tools Page) I should be able to click the inputs here
  > The inputs are only enabled when the wallet is connected

### CODE REVIEW

File: site/components/layout/HomeFooter.js
Comment (ArturDolzan) [inline site/components/layout/HomeFooter.js:12]: I think we can remove this div
```suggestion
      <p>{t('copyright')}</p>
```

File: site/components/layout/ToolsFooter.js
Comment (ArturDolzan) [inline site/components/layout/ToolsFooter.js:10]: Same here

**Review by ArturDolzan — COMMENTED** [review]
Two minor comments otherwise LGTM

**Comment by gndelia** [comment]
Hey there! Can we get a few screenshots to better see the visual updates?

File: site/components/layout/HomeFooter.js
Comment (gndelia) [inline site/components/layout/HomeFooter.js:14]: Next'js `Link` should be used for relative links. For external links (As are all on this page), an anchor tag (`a`) should be used.

It is also a good practice to add `rel="noopener noreferrer"`, though depending on the source you check, one of both may be redundant (or maybe one is not recognized by certain browsers). So playing safe I'd just add both

File: site/components/layout/ToolsFooter.js
Comment (gndelia) [inline site/components/layout/ToolsFooter.js:37]: Same here re: links, and if the list of links and SVG and colors are exactly the same as above, it seems they could be moved into a "socials" component or something like that

File: site/components/layout/HomeFooter.js
Comment (gndelia) [inline site/components/layout/HomeFooter.js:32]: About these links, I notice they are already used in the portal, and probably in some other places (like pop miner, and the editor maybe?).

I created a [repository](https://github.com/hemilabs/hemi-socials) to export them. If I get to publish them before this PR gets merged, you could update them. It will be similar to [this](https://github.com/hemilabs/ui-monorepo/tree/main/hemi-socials)

File: site/components/layout/HomeFooter.js
Comment (alvarengathomas) [reply site/components/layout/HomeFooter.js]: Fixed

File: site/components/layout/ToolsFooter.js
Comment (alvarengathomas) [reply site/components/layout/ToolsFooter.js]: Fixed

File: site/components/layout/HomeFooter.js
Comment (alvarengathomas) [reply site/components/layout/HomeFooter.js]: Fixed

File: site/components/layout/ToolsFooter.js
Comment (alvarengathomas) [reply site/components/layout/ToolsFooter.js]: Fixed

**Comment by alvarengathomas** [comment]
> Hey there! Can we get a few screenshots to better see the visual updates?

There is no much difference in the visuals, just minor details as you can see below:

Landing Page (Before this PR):
old-landing-page

Landing Page (After this PR):
new-landing-page

Tools Page (Before this PR):
old-tools-page

Tools Page (After this PR):
new-tools-page

File: site/components/layout/HomeFooter.js
Comment (alvarengathomas) [reply site/components/layout/HomeFooter.js]: I think it's a good idea, but we can create a full library with all shared components used in all dapps.

File: site/components/layout/Socials.js
Comment (gndelia) [inline site/components/layout/Socials.js:11]: @rmilrad  should we add Youtube? I think it is on all the other pages (it's in the portal and the main website).

Similar question about Telegram links (though it is not in the portal)

Comment (rafappelt) [reply site/components/layout/Socials.js:11]: This kind of change shouldn't be required on another task?
I mean, the PR is correct according the Issue and its approved layout.

**Review by rafappelt — APPROVED** [review]

File: site/components/layout/Socials.js
Comment (gndelia) [reply site/components/layout/Socials.js:11]: @rafappelt sure, it can be done in another PR, I'm not against it. But I'm just asking Ryan 😄  Thomas can feel free to merge the PR and create an issue if desired

Comment (alvarengathomas) [reply site/components/layout/Socials.js:11]: The PR can only be merged if @gabmontes (code owner) approves

**Review by gabmontes — APPROVED** [review]
@alvarengathomas please replace the social links with [hemi-socials](https://github.com/hemilabs/hemi-socials) in a follow-up PR, so we don't block this one.

**Pull request merged**
